src/renderers/shaders/ShaderChunk/shadowmap_vertex.glsl

```diff
@@ -2,7 +2,36 @@
 
 	for( int i = 0; i < MAX_SHADOWS; i ++ ) {
 
-		vShadowCoord[ i ] = shadowMatrix[ i ] * worldPosition;
+		#if defined(POINT_LIGHT_SHADOWS)
+
+			// if shadowDarkness[ i ] < 0.0, that means we have a point light with a cube
+			// shadow map
+			if( shadowDarkness[ i ] < 0.0 ) {
+
+				// When we have a point light, the @shadowMatrix uniform is used to store
+				// the inverse of the view matrix, so that we can get the world-space
+				// position of the light.
+
```

Comment:
Yes. Sorry for not catching that one. In fact, you can find the same trick in the test for the software cubemapper transforming the ray. Inverse, of course...

Since we assume view <-> world to be orthonormal, we can even go without the normalization - just multiply from the other side.